link `at` and `with` together (#29099)
mdn/content: files/en-us/web/javascript/reference/global_objects/array/with/index.md

The `with()` method changes the value of a given index in the array, returning a new array with the element at the given index replaced with the given value. The original array is not modified. This allows you to chain array methods while doing manipulations.

> By combining this with {{jsxref("Array.prototype.at()", "at()")}}, you can both write and read (respectively) the array with negative indices.

Reviewer: ```suggestion
By combining `with()` with {{jsxref("Arra/at", "at()")}}, you can both write and read (respectively) an array using negative indices.
```
